XMonad/Layout/WindowNavigation.hs

```diff
@@ -72,6 +72,14 @@ data Navigate = Go Direction2D | Swap Direction2D | Move Direction2D
               | Apply (Window -> X()) Direction2D -- ^ Apply action with destination window
 instance Message Navigate
 
+-- | Used with @configurableNavigation@ to specify how to show reachable windows'
```

Comment:
AFAIK, if we want to link to functions we have to use `'…'` instead of `@…@` (which is just monospace)

Reply:
That's what I thought, but the existing markup uses `@…@`. I'll fix that too.

Reply:
That's what I thought, but the existing markup uses `@…@`. I'll fix that too.